apps/bfd-data-npi/src/main/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookup.java

```diff
@@ -48,23 +49,31 @@ public static NPIOrgLookup createNpiOrgLookupForTesting() throws IOException {
    * Factory method for creating a {@link NPIOrgLookup } for production that does not include the
    * fake org name.
    *
+   * @throws IOException if there is an issue reading file
    * @return the {@link NPIOrgLookup }
    */
   public static NPIOrgLookup createNpiOrgLookupForProduction() throws IOException {
     if (npiOrgLookupForProduction == null) {
-      npiOrgLookupForProduction = new NPIOrgLookup(false);
+      InputStream npiDataStream = getFileInputStream(App.NPI_RESOURCE);
+      npiOrgLookupForProduction = new NPIOrgLookup(npiDataStream);
     }
 
     return npiOrgLookupForProduction;
   }
 
+  /** Constructs an {@link NPIOrgLookup}. */
+  protected NPIOrgLookup() {
+    npiOrgHashMap.put(FAKE_NPI_NUMBER, FAKE_NPI_ORG_NAME);
+  }
+
```

Comment:
Probably want to mention this constructor is for testing purposes in the documentation.